db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
```

Comment:
I don't understand whats the difference between these two options.
Anyway I couldn't add the character ' in this string.